modules/dataproc/README.md

```diff
@@ -46,6 +46,35 @@ module "processing-dp-cluster" {
 # tftest modules=1 resources=1
 ```
 
+### Cluster with CMEK encrypotion
+
+To set cluster configuration use the Customer Managed Encryption key, set '' variable. The Compute Engine service agent and the Cloud Storage service agent needs to have 'CryptoKey Encrypter/Decrypter' role on they configured KMS key ([Documentation](https://cloud.google.com/dataproc/docs/concepts/configuring-clusters/customer-managed-encryption)).
```

Comment:
Lore, code quoting is via vackticks not single quotes. What's the meaning of the `set '' variable`?

Reply:
`and the Cloud Storage service agent needs` ---> `and the Cloud Storage service agent need`

Reply:
Done